addons/mail/static/src/components/thread_view_topbar/thread_view_topbar.xml

```diff
@@ -23,11 +23,23 @@
                             Discuss
                         </div>
                     </t>
-                    <t t-if="threadViewTopBar.thread and threadViewTopBar.thread.description">
+                    <t t-if="threadViewTopBar.thread and threadViewTopBar.thread.isChannelDescriptionChangeable">
```

Comment:
I don't think this condition needs to be changed.

Reply:
@seb-odoo It is now because if there is no description, I add a "Add a description" label. So I need to know if it can have description, not if there is currently a description.